src/engine/sim_engine/pool.rs

```diff
@@ -112,6 +112,16 @@ impl Pool for SimPool {
         paths: &[&Path],
         tier: BlockDevTier,
     ) -> StratisResult<SetCreateAction<DevUuid>> {
+        if paths.is_empty() {
+            return if !self.has_cache() && tier == BlockDevTier::Cache {
+                Err(StratisError::Engine(
+                    ErrorEnum::Invalid,
+                    "At least one blockdev path is required to add blockdevs".to_string(),
```

Comment:
"At least one blockdev path is required to initialize a cache." would be better, and matches pool_create error. This also applies to ```StratEngine::add_blockdevs``` below.